Library/Formula/check.rb

```diff
@@ -1,6 +1,6 @@
 class Check < Formula
   desc "C unit testing framework"
-  homepage "http://check.sourceforge.net/"
+  homepage "http://libcheck.github.io/check/"
```

Comment:
Mind making this HTTPS? Thanks! http://bot.brew.sh/job/Homebrew%20Pull%20Requests/41287/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_check/